ui/ui-components/pages/model/[modelId]/overview.tsx

```diff
@@ -171,12 +171,15 @@ export const getServerSideProps: GetServerSideProps<Props> =
       `sampleRecord:${grouparooUiEdition()}`
     )?.[modelId as string];
 
-    const sampleRecord = sampleRecordId
-      ? await client.request<Actions.RecordView>(
-          "get",
-          `/record/${sampleRecordId}`
-        )
-      : null;
+    let sampleRecord: Actions.RecordView = null;
+    try {
+      sampleRecord = sampleRecordId
+        ? await client.request<Actions.RecordView>(
+            "get",
+            `/record/${sampleRecordId}`
+          )
+        : null;
+    } catch (err) {}
```

Comment:
Is this the correct way to catch and ignore client errors in the server context? I wasn't able to achieve the same thing with setting an errorhandler

Reply:
Yep! There isn't an error-handler available when hydrating.  You might want to log something just in case